### DISCUSSION TITLE
小建议：自动按字母顺序排序

### DISCUSSION THREAD

**Discussion by jwen15** [tuna/thuthesis#690] [Q&A] [answered]
## 编译环境
编译的系统： Windows 10
TeX 发行版：TeX Live 2021
模板版本：v7.2.3
模板类型：doctor

## 描述问题
有个小建议，***符号和缩略语说明***，能不能加一个功能，就是***自动按字母顺序排序***？

虽然《清华大学研究生学位论文写作指南》中并没有这个要求，但是感觉排序之后才更方便查找符号和缩略语。

如果不好搞就算了。打扰了！

**Comment by zepinglee** [comment, marked as answer]
用 `nomencl` 的话默认会自动排序的，参考文档和示例。
